proxyllm/proxyllm.py

```diff
@@ -219,7 +219,7 @@ def _setup_user_models(
         ) from e
 
 
-def _setup_models_cost_data(settings: List[Dict[str, Any]]) -> Dict[str, Any]:
+def _setup_model_data(settings: List[Dict[str, Any]]) -> Dict[str, Any]:
```

Comment:
repurposed function to get model data from yml for both cost and proficiency routing